server/routes/varyingLicences/handlers/timeline.ts

```diff
@@ -24,10 +26,14 @@ export default class TimelineRoutes {
     if (licence.statusCode === VARIATION_APPROVED) {
       return PRINT_TO_ACTIVATE
     }
-    if (licence.statusCode === ACTIVE && !licence.isReviewNeeded) {
+    if (licence.statusCode === ACTIVE && !licence.isReviewNeeded && licence.kind !== LicenceKind.HDC) {
       return VIEW_OR_VARY
     }
 
+    if (licence.statusCode === ACTIVE && !licence.isReviewNeeded && licence.kind === LicenceKind.HDC) {
+      return VIEW
+    }
```

Comment:
👍  
(me and @SebNegahbanMoJ hate this file 😂)

Reply:
You can add me to that list too.